ingestors/maven/maven_central.go

```diff
@@ -0,0 +1,34 @@
+package maven
```

Comment:
I think I'd just leave this all in the ingestors package. No need to get all deeply nested with it I think?

Reply:
We'll be adding 5 or 6 more at least. That's why I went for it. I can move it back though.

Reply:
I think as long as they're all mave_yada_yada it's easy enough to find em? Just felt really odd to have the `maven.NewMavenCentral()` below?

Reply:
I'm not strongly opinionated, but Effective Go [says](https://golang.org/doc/effective_go.html#package-names): `Another convention is that the package name is the base name of its source directory; the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64. `

Reply:
Yea true, it's just gonna get messy in there. I don't really know what the right answer is on the maven stuffs. Perhaps what @mellisio did originally is more sensible 🤷